CHANGELOG.md

```diff
@@ -2,13 +2,24 @@
 
 ## Unreleased
 
+## 1.1.0 - 2024-01-10
+
 Full list of differences found in [this compare](https://github.com/open-telemetry/opentelemetry-proto/compare/v1.0.0...main).
 
 ### Added
 
 * Add a field for W3C-specified Trace Context flags to the `Span` and `Link`.
   [#503](https://github.com/open-telemetry/opentelemetry-proto/pull/503)
 
+### Changed
+
+* Update and fix OTLP JSON examples.
+  * [#516](https://github.com/open-telemetry/opentelemetry-proto/pull/516)
+  * [#510](https://github.com/open-telemetry/opentelemetry-proto/pull/510)
+  * [#499](https://github.com/open-telemetry/opentelemetry-proto/pull/499)
```

Comment:
```suggestion
  [#516](https://github.com/open-telemetry/opentelemetry-proto/pull/516)
  [#510](https://github.com/open-telemetry/opentelemetry-proto/pull/510)
  [#499](https://github.com/open-telemetry/opentelemetry-proto/pull/499)
```

Reply:
Align with existing practice, e.g. https://github.com/open-telemetry/opentelemetry-proto/blob/main/CHANGELOG.md#maturity-1.

Reply:
I've done it, was trolling a bit sorry :)